frontend/src/documents/DocumentPreview.svelte

```diff
@@ -25,7 +25,7 @@
 
 {#if extension === "pdf"}
   <object title={filename} data={url} />
-{:else if ["csv", "txt"].includes(extension)}
+{:else if ["csv", "txt", "qfx"].includes(extension)}
```

Comment:
Now that this list gets longer, I think I would prefer to move it to the top as a `plainTextExtensions` next to `imageExtensions`